sdk/communication/communication-networktraversal/package.json

```diff
@@ -0,0 +1,137 @@
+    "execute:js-samples": "dev-tool samples run dist-samples/javascript",
+    "execute:ts-samples": "dev-tool samples run dist-samples/typescript/dist/dist-samples/typescript/src/",
+    "extract-api": "tsc -p . && api-extractor run --local",
+    "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
+    "integration-test:browser": "karma start --single-run",
+    "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --full-trace -t 300000  dist-esm/test/public/*.spec.js dist-esm/test/public/node/*.spec.js",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "lint:fix": "eslint package.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
+    "lint": "eslint package.json api-extractor.json src test --ext .ts",
+    "pack": "npm pack 2>&1",
+    "prebuild": "npm run clean",
+    "test": "npm run build:test && npm run unit-test && npm run integration-test",
+    "test:browser": "npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
+    "test:node": "npm run build:test && npm run unit-test:node && npm run integration-test:node",
+    "test:watch": "npm run test -- --watch --reporter min",
+    "unit-test:browser": "karma start --single-run",
+    "unit-test:node": "mocha --reporter ../../../common/tools/mocha-multi-reporter.js dist-test/index.node.js",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser",
+    "docs": "typedoc --excludePrivate --excludeNotExported --excludeExternals --stripInternal --mode file --out ./dist/docs ./src"
+  },
+  "//sampleConfiguration": {
+    "productName": "Azure Communication Services - Network Traversal",
+    "productSlugs": [
+      "azure",
+      "azure-communication-services"
+    ],
+    "requiredResources": {
+      "Azure Communication Services account": "https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource"
+    }
+  },
+  "files": [
+    "dist/",
+    "dist-esm/src/",
+    "types/communication-networktraversal.d.ts",
+    "README.md",
+    "LICENSE"
+  ],
+  "keywords": [
+    "azure",
+    "cloud",
+    "communication"
+  ],
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/communication/",
+  "repository": "github:Azure/azure-sdk-for-js",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "sideEffects": false,
+  "prettier": "@azure/eslint-plugin-azure-sdk/prettier.json",
+  "dependencies": {
+    "@azure/abort-controller": "^1.0.0",
+    "@azure/communication-common": "^1.0.0",
+    "@azure/communication-identity": "^1.0.1",
```

Comment:
is there any particular reason to be dependent on 1.0.1 instead of 1.0.0?

Reply:
Also, can this be a dev dependency? I don't see outside of tests/samples where the identity package is used
(please see chat as an example)